VCFv4.4.draft.tex

```diff
@@ -580,16 +580,9 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 \end{samepage}
-This key can be derived from the REF/ALT fields but is useful for filtering.
-The reserved values must be used for the types listed below:
-\begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
-  \item BND: Breakend
-\end{itemize}
+
+This field has been deprecated due to redundancy with ALT.
+Refer to the ALT field for set of valid symbolic structural variant alleles.
```

Comment:
```suggestion
This field has been deprecated due to redundancy with ALT.
Refer to the ALT field for the set of valid symbolic structural variant alleles.
```

Reply:
Dumb question: Does “ALT field” here refer to column 5 (so it's talking about breakend notation and/or…?) or is it talking about an INFO or FORMAT tag (possibly newly defined in another PR? If so, which?)?

Reply:
@d-cameron to confirm, but I understand this to refer specifically to column 5. Indeed, it always contains either the variation type (DEL/DUP/INV/...), or the breakend notation, which makes it possible to always infer the type

Reply:
Ah, okay. That would be clearer with a cross-reference to §1.4.5 (“Alternative allele field format”).

Reply:
Valid point, @d-cameron do you think you could add it to this PR as well?